lib/escape.js

```diff
@@ -6,7 +6,16 @@ exports.escapeXML = function escapeXML (s) {
     .replace(/</g, '&lt;')
     .replace(/>/g, '&gt;')
     .replace(/"/g, '&quot;')
-    .replace(/"/g, '&apos;')
+    .replace(/'/g, '&apos;')
+}
+
+exports.unescapeXML = function unescapeXML (s) {
+  return s
+    .replace(/\&(amp|#38);/g, '&')
+    .replace(/\&(lt|#60);/g, '<')
+    .replace(/\&(gt|#62);/g, '>')
+    .replace(/\&(quot|#34);/g, '"')
+    .replace(/\&(apos|#39);/g, "'")
```

Comment:
just moved unescapeXML function up

@lloydwatkin why do we need to match # codes ?